config/config.go

```diff
@@ -67,18 +67,18 @@ type AuthConfiguration struct {
 }
 
 type StrategyConfiguration struct {
-	Type    StrategyProvider             `json:"type"`
-	Default DefaultStrategyConfiguration `json:"default"`
+	Type    StrategyProvider             `json:"type" valid:"required~please provide a valid strategy type,in(default)~unsupported strategy type"`
```

Comment:
@Dotunj This looks like we're coupling configuration validation with server validation. I don't think we want to do this.

Reply:
@Subomi Are you referring to just this particular field?